server/src/metakb/harvesters/base.py

```diff
@@ -54,18 +55,20 @@ def harvest(self) -> _HarvestedData:
     def save_harvested_data_to_file(
         self, harvested_data: _HarvestedData, harvested_filepath: Path | None = None
     ) -> bool:
-        """Save harvested data to JSON file
+        """Save harvested data to JSON file.
 
         :param harvested_data: harvested data from a source
-        :param harvested_filepath: Path to the JSON file where the harvested data will
-            be stored. If not provided, will use the default path of
-            ``<APP_ROOT>/data/<src_name>/harvester/<src_name>_harvester_YYYYMMDD.json``
+        :param harvested_filepath: Path to the JSON file location where the harvested data
+            will be stored. If not provided, will use the default path of
+            ``<METAKB_DATA_ROOT>/<src_name>/harvester/<src_name>_harvester_YYYYMMDD.json``,
```

Comment:
Should `METAKB_DATA_ROOT` be `METAKB_DATA_DIR`?

Reply:
Oh yeah, I saw this comment and it made me think a bit more about what I was trying to say with these docstrings... will tweak a bit